splinterd/tests/framework/network.rs

```diff
@@ -56,6 +60,17 @@ impl Network {
             let temp_dir = TempDir::new("scabbard_data")
                 .map_err(|e| InternalError::from_source(Box::new(e)))?;
 
+            // Create a store factory to allow for the Biome endpoints to be used
+            let store_factory = Box::new(MemoryStoreFactory::new()?);
+            // Build the `BiomeResourceProvider` to allow the node to access `Biome` endpoints
+            let mut biome_resource_provider = BiomeResourceProviderBuilder::default()
```

Comment:
These should be with-in the node some how.  Probably we should have a `with_biome_enabled` function (for now) on the Node builder, and then they are constructed internally at the appropriate time.

Reply:
I changed this to add the `with_biome_enabled` to the `NodeBuilder` struct and since the resources that are built need access to the store factory, we can internally construct the resources in the `RunnableNode` module.